scopes/react/react-native/react-native.docs.md

```diff
@@ -0,0 +1,345 @@
+---
+labels: ['react native', 'typescript', 'environments', 'extension']
+description: 'A React Native extension'
+---
+
+The React Native environment is an extension of the [React environment](https://bit.dev/teambit/react/react). It is a one-stop-shop for React Native components in a Bit workspace. It uses various services, provided by other extensions, to handle the life events of React components, managed in a Bit workspace.
+
+The React Native environment spares you the overhead of setting up your own React environment and creates a standardized and shareable development environment for you and your team.
+
+#### Features
+
+- **Blazing fast environment setup**: Setting up the React Native environment requires nothing more than configuring a workspace to use a Bit extension.
+  Get started in seconds and focus on the thing that matters most - delivering great features.
+
+- **Less to learn**: Using the React Native environment means you don't have to get into all the details of your build tooling.
+  That's a especially important when onboarding a new developer to your team.
+
+- **Customizable and extensible**: React Native can be extended to add or override configurations.
+  Quickly add your own modifications to get a React Native environment "flavour" that best suits your needs.
+  Export your React Native environment extension to a remote scope to have it available to all your team.
+
+- **Standardized development**: Use the React environment to maintain consistency in development across multiple decoupled Bit workspaces.
+
+- **Easy to maintain**: Get React's latest updates with just a simple `bit import` command. Roll-back as easily, if needed.
+
+## Quickstart & configuration
+
+> To use the React environment, set it in the `workspace.jsonc` configuration file. React can only be configured using the 'variants' config API.
+
+### Use React as the default environment
+
+Apply the React environment on all components in the workspace, using the wildcard character `*`.
+
+```json
+{
+  "teambit.workspace/variants": {
+    "*": {
+      "teambit.react/react-native": {}
+    }
+  }
+}
+```
+
+### Use React on a specific group of components
+
+Apply the React environment on a limited set of components. For example, all components inside the `components/react-ui` directory.
+
+```json
+{
+    "teambit.workspace/variants": {
+    "*": {
+        "teambit.react/react-native": {}
+        }
+    }
+}
+}
+
+```
+
+_Learn more about configuring a selected set of components, [here](https://bit.dev/teambit/workspace/variants)_
+
+### Set the React environment to compile MDX components
+
+The React environment has built-in support for MDX.
+
+To set React to compile MDX components (so that they can be shared as a distributable code), set the `mdx` property to `true`.
+
+```json
+{
+  "teambit.workspace/variants": {
+    "*": {
+      "teambit.react/react-native": {
+        "mdx": true
+      }
```

Comment:
I would delete this example, it won't be like this. it will be configured via env override / use api